starknet/src/space/space.cairo

```diff
@@ -664,14 +665,26 @@ mod Space {
             i += 1;
         };
         _active_voting_strategies::write(cachedActiveVotingStrategies);
+        let mut i = 0_usize;
         _next_voting_strategy_index::write(cachedNextVotingStrategyIndex);
     }
-    // TODO: need to impl set_bit to false first
-    // fn _remove_voting_strategies(_voting_strategies: Array<Strategy>) {
-    //     let index = next_voting_strategy_index::read();
-    //     voting_strategies::write(index, _voting_strategy);
-    //     next_voting_strategy_index::write(index + 1_u8);
-    // }
+
+    fn _remove_voting_strategies(_voting_strategies: Array<u8>) {
```

Comment:
we should probably add a check for at least one voting strategy like https://github.com/snapshot-labs/sx-evm/blob/6c7830f9466ad4f6324afa6b4e02fc3818791162/src/Space.sol#L384C5-L384C5

Reply:
07a8bb94fb0a12261d6e6f270878f10dd2e86833